### PR TITLE
Fix Tower-Pebble documentation

### LINKED ISSUE #181: Invalid Tower-Pebble link
The link to the Tower-Pebble **Android companion app** in the [Android DroneKit documentation](http://collabedit.com/b59c7) is invalid.

### LINKED ISSUE #182: Update 3DR Services app icon in the Tower-Pebble DroneKit documentation

### LINKED ISSUE #183: Tower-Pebble guide review
The Tower-Pebble guide is live on the [Android DroneKit site](http://android.dronekit.io/pebble_app.html).
Comments and feedback welcomed!

### PR DESCRIPTION
**Pull request by squilter**
Don't merge until we have feedback from #183 

Fixes #181
Fixes #182 
Fixes #183

### CODE REVIEW

**Comment by djmedina** [comment]
Awesome work, everything looks great. It seems like we are assuming that our audience has some experience developing watch applications, which is a valid assumption to make.

This bullet list looks a little off
Imgur

I would add more spacing between sections. This comment also applies to the Hello Drone tutorial
Imgur

This link is dead:
[PebbleCommunicatorService](https://github.com/DroidPlanner/tower-pebble/blob/master/Android/src/main/java/org/droidplanner/Pebble/PebbleCommunicatorService.java)

We should have single formatting for flight modes. I see modes between quotes, and some other times in a different font.

Just a _comment_, but do we really want to start by pulling an existing project from github, wouldn't it be more useful if we start with a blank project?

**Comment by squilter** [comment]
@djmedina thanks for the feedback.

The link is fixed now.  I will fix the flight mode formatting.

I'm not sure how to fix the bullet list.  It might be in the sphinx theme.

The space between sections is definitely in the sphinx theme.  When I compile the docs with an older version of the sphinx theme, it looks like this:
selection_012

**Comment by squilter** [comment]
@ne0fhyk I think I have fixed all parts that I am able to fix.  Some of the small formatting things that Diego pointed out are problems in the sphinx theme.

**Comment by hamishwillee** [comment]
@squilter Nice job. The code blocks should probably be indented too, as part of the bulleted lists. The image below shows how it is, and how it should be. This is controlled in sphinx by indentation:

image

Yes, the heading level stuff is a problem in the theme. I assume you fixed the other bullet stuff, but let me know if it doesn't build OK and I can look at it.

**Comment by squilter** [comment]
@hamishwillee The only problem left is the list formatting that Diego identified.  Do you know what to do about that?

**Comment by squilter** [comment]
Here is feedback from Kaitlyn

- [ ] Dividing up the text for "Project setup" into two sub sections:
        Clone Tower-Pebble repo
        Import project
- [ ] Changing the text "Understanding the Watchapp Code" to "Watchapp Code Overview" and dividing up the text into two sub sections:
        Parts
        Code overview -- Can you make the two main bullets about the code overview (Listens for two strings, Sends requests when the user pushes buttons) not bullets, and just strings? I'll fix the styles for the nesting of the list elements so that the bullets populate correctly, but I don't think that the two main overview points need to be bullets themselves. 
- [ ] Change the text "Understanding the Android Code" to "Android Code Overview" and divide up the text into two sections:
        Functionality
        Handling Requests 
- [ ] Expand on the "Summary" portion haha -- Can you make up a brief paragraph about what the user can use all this new info on the Pebble stuff for? Maybe a pat on the back for reading everything on the page? Maybe we could add a link for questions regarding the pebble documentation? Just throwing stuff out there.

**Comment by hamishwillee** [comment]
@squilter 

>The only problem left is the list formatting that Diego identified. Do you know what to do about that?

I prefer not to use the "definition list" format with the text of the top level bullet point bold. While this should work, the behaviour is somewhat "undefined". 

Please change text to as below  (line between bullets, indent second bullets as shown):
```
* Listens for two strings: **mode** and **telem**:

  * **mode** is displayed in bold at the top of the screen
  * **telem** is displayed at the bottom of the screen

* Sends requests when the user pushes buttons:

  * ``REQUEST_MODE_FOLLOW`` when the top button is pushed (if the mode is not already ``follow``)
  * ``REQUEST_CYCLE_FOLLOW_TYPE`` when the top button is pushed (if  the mode is already ``follow``)
  * ``REQUEST_PAUSE`` when the middle button is pushed
  * ``REQUEST_MODE_RTL`` when the bottom button is pushed
```

To fix the other problem [I identified above](https://github.com/DroidPlanner/DroneKit-Android/pull/184#issuecomment-119036888) you need to change the text to look like this (only shown for part of two bulletsbullet, but you need to roll out to all the points):

```
1. :file:`PebbleCommunicatorService` needs to implement :file:`DroneListener`:

   .. code-block:: java

       public class PebbleCommunicatorService extends Service implements DroneListener, Towerlistener{

2. The :file:`DroneListener` needs to be registered:

   .. code-block:: java
       :emphasize-lines: 6

       //Runs when 3dr-services is connected.  Immediately connects to drone.
       @Override
       public void onTowerConnected() {
       if (!drone.isStarted()) {
```

Basically you use three spaces to align the code block to the bullet, then four space indent for the actual code (after the three space indent)

Make sense?

**Comment by squilter** [comment]
Okay, I think I'm done.  Can we merge?

**Comment by ne0fhyk** [comment]
Gladly!

**Comment by hamishwillee** [comment]
@squilter A bit of a trudge, but well done!
